Modelica/Electrical/Analog/Examples/Lines/CompareCmatrix.mo

```diff
@@ -0,0 +1,273 @@
+  connect(c20.n, ground1.p)
+    annotation (Line(points={{-54,-32},{-54,-36},{-18,-36}}, color={0,0,255}));
+  connect(c10.n, ground1.p)
+    annotation (Line(points={{-66,22},{-66,-36},{-18,-36}}, color={0,0,255}));
+  connect(c10.p, c12.p)
+    annotation (Line(points={{-66,42},{-66,52},{-18,52}}, color={0,0,255}));
+  connect(c12.n, c23.p)
+    annotation (Line(points={{-18,32},{-18,18}}, color={0,0,255}));
+  connect(v1a.p, v2a.p) annotation (Line(points={{-124,48},{-132,48},{-132,18},{
+          -124,18}}, color={0,0,255}));
+  connect(v2a.p, v3a.p) annotation (Line(points={{-124,18},{-132,18},{-132,-2},
+          {-124,-2}},color={0,0,255}));
+  connect(c12.p, c13.p)
+    annotation (Line(points={{-18,52},{0,52},{0,32}},     color={0,0,255}));
+  connect(c13.n, c23.n)
+    annotation (Line(points={{0,12},{0,-2},{-18,-2}},     color={0,0,255}));
+  connect(v1a.n, r1n.p)
+    annotation (Line(points={{-104,48},{-96,48}}, color={0,0,255}));
+  connect(r1n.n, c12.p) annotation (Line(points={{-80,48},{-66,48},{-66,52},{
+          -18,52}}, color={0,0,255}));
+  connect(v3a.n, r3n.p)
+    annotation (Line(points={{-104,-2},{-96,-2}}, color={0,0,255}));
+  connect(v2a.n, r2n.p)
+    annotation (Line(points={{-104,18},{-96,18}}, color={0,0,255}));
+  connect(r2n.n, c23.p) annotation (Line(points={{-80,18},{-40,18},{-40,24},{
+          -18,24},{-18,18}},
+                         color={0,0,255}));
+  connect(r3n.n, c23.n)
+    annotation (Line(points={{-80,-2},{-18,-2}}, color={0,0,255}));
+  connect(c10.n, c20.n) annotation (Line(points={{-66,22},{-66,-36},{-54,-36},
+          {-54,-32}}, color={0,0,255}));
+  connect(ground1.p, v3a.p) annotation (Line(points={{-18,-36},{-132,-36},{
+          -132,-2},{-124,-2}}, color={0,0,255}));
+  connect(ground.p, v3.p) annotation (Line(points={{86,-32},{20,-32},{20,-12},
+          {28,-12}}, color={0,0,255}));
+  connect(c20.p, c23.p) annotation (Line(points={{-54,-12},{-54,18},{-40,18},
+          {-40,24},{-18,24},{-18,18}}, color={0,0,255}));
+  connect(c30.p, c23.n)
+    annotation (Line(points={{-36,-12},{-36,-2},{-18,-2}}, color={0,0,255}));
+  annotation (
+    Diagram(coordinateSystem(extent={{-140,-60},{120,60}})),
+    Documentation(info="<html>
+<p>MSL uses, inside M_OLine, a network of physical capacitors to model capacitive effects across conductors.</p>
+<p>Insstead, in typical PowerSystems approach, a capacitance matrix is used, corresponding to the vector equation:</p>
+<p><b>i</b>=<i><b>C&nbsp;</b></i>d<b>v</b>/dt&nbsp;</p>
+<p>where</p>
+<p>- <i><b>i</b></i> is the vector of currents entering a multiple-conductor lines (n conductors with a return wire gives rise to n currents)</p>
+<p>- <i><b>v</b></i> is the vector of voltages at a port of a multiple-conductor lines (n conductors with a return wire gives rise to n voltages)</p>
+<p>The <i><b>C</b></i> matrix is symmetrical, and therefore it has n(n+1) independent terms. the diagonal terms of this matrix are negative.</p>
```

Comment:
I used \<strong\> as in `Modelica.Math.Matrices`
However I used also \<em\> since IEC requires for physical quantities such as currents, voltages, and capacitances italic style.
So, for generic math, we must just to embold by \<strong\>; but for math between physical quantities we should also italicise by \<em\>.

Is this acceptable for you?

Reply:
The question is, whether there is a consensus on using IEC within MSL. I don't believe so. You can open a new issue on it. 
For now, IMO the library officers shall decide: @christiankral @AHaumer @christophclauss

Reply:
> The question is, whether there is a consensus on using IEC within MSL. I don't believe so.
 
I did not expect this!

However, it seems to me that writing C in bold-italic does not break any MSL rule, while being IEC compliant.  I mean, it is compatible with `Modelica.Math.Matrices` writing:

_when I have a matrix composed by generic numbers I just use bold; when that matrix is composed by numbers referring to physical quantities, use bold  and italic._ 

AFAIK this would be compliant with IEC without going against `Modelica.Math.Matrices`  writing of documentation.

Moreover,  `Modelica.Math.Matrices` is not that consistent. For instance in solve2 description matrix names are written in bold, while in leastSquares', I read "Returns a solution of equation A*x = b in a least square sense" (without A, x, b being bold). 
So, it MSL is a bit relaxed on how to manage these things (and I agree with this approach).

Additional note. My purpose is giving users something useful, not changing the world. If someone tells me that capacitance matrix must not be italic, otherwise the PR cannot be merged, I will remove \<em\>, even though for me it is not the best thing to do.

Reply:
and... 
it is not just IEC (and ISO) that use italic for physcical quantities, but, AFAIK, **all technical standardisation bodies**.

The BIPM SI-Brochure-9-EN available from
https://www.bipm.org/en/publications/si-brochure

In that document all physical quantities are written in italic, invariably.

It is the units of measure that must not be italic, even if a lot of writers don't know this (I've found this mistake in many, many papers, even in the most authoritative journals).